include/eve/detail/function/simd/arm/neon/movemask.hpp

```diff
@@ -0,0 +1,116 @@
+//==================================================================================================
+/**
+  EVE - Expressive Vector Engine
+  Copyright 2021 Joel FALCOU
+  Copyright 2021 Jean-Thierry LAPRESTE
+  Copyright 2021 Dens YAROSHEVKIY
+
+  Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+  SPDX-License-Identifier: MIT
+**/
+//==================================================================================================
+#pragma once
+
+#include <eve/arch/logical.hpp>
+#include <eve/detail/meta.hpp>
+#include <eve/function/convert.hpp>
+
+#include <bit>
+
+namespace eve::detail
+{
+  template<typename T>
+  EVE_FORCEINLINE std::uint32_t every_2nd_byte_arm64(T const& v) noexcept
+  {
+    using u8_8  = wide<std::uint8_t,  eve::fixed<8>, arm_64_>;
```

Comment:
I am rather unconfortable by instanciating wide like that. This was historically a source of longer compile time.
This can be set as `typename T::template rebase<stduint8_t,fixed<8>>;`

Reply:
Done